Cargo.toml

```diff
@@ -40,9 +40,12 @@ libc = "0.2"
 
 [dev-dependencies]
 colored = "2"
-ui_test = "0.1"
+ui_test = "0.2"
 # Features chosen to match those required by env_logger, to avoid rebuilds
-regex = { version = "1.5.5", default-features = false, features = ["perf", "std"] }
+regex = { version = "1.5.5", default-features = false, features = [
+    "perf",
+    "std",
+] }
```

Comment:
Is this some kind of auto-formatter? I don't think I like how it formats this.

Reply:
ugh... I just wanted syntax highlighting and now it's doing this everywhere

Reply:
format-on-save considered harmful. ;)